### DISCUSSION TITLE
Removing password for open network

### DISCUSSION THREAD

**Discussion by SolarDiode** [RaspAP/raspap-webgui#975] [Q&A] [answered]
How am I able to remove the password to allow the network to be open? When I do this it defaults back the ChangeMe?
Thanks

**Comment by billz** [comment, marked as answer]
On the Hotspot > Security tab, select 'None' for Security type then restart your AP.

**Reply by lear0203** [reply]
Hallo,

ich habe auch noch eine Frage zum Passwort.
Wo kann ich die Länge des Passworts einstellen, die RaspAP akzeptiert.
Ich hätte es gern, dass der AP nur Passwörter mit der Mindestlänge von 25 Zeichen akzeptiert.

Desweiteren würde ich Sonderzeichen auch in die Passwortvorschrift aufnehmen.

Vielen Dank

**Reply by billz** [reply]
The passphrase rules are based on [IEEE standard 802.11i-2004](https://en.wikipedia.org/wiki/IEEE_802.11i-2004). You may modify the length requirement if you wish by changing lines 217-218 of the code below.

https://github.com/RaspAP/raspap-webgui/blob/c24104008a4559df7da39d325e0eb30d0b0510b7/includes/hostapd.php#L215-L223

Re: special characters, again per the IEEE spec only ASCII characters from 32 to 126 are permitted. For this reason `ctype_print()` is used. Therefore, for example, the Euro sign "€", German "ä" and British pound symbol "£" fall outside this range.
